apps/gallery/js/info.js

```diff
@@ -52,10 +52,30 @@ function showFileInformation(fileinfo) {
 
       // Populate info overlay view
       MediaUtils.populateMediaInfo(data);
+      // Hide Resolution for video files. See Bug 1217989
+      setFieldVisibility('info-resolution', fileinfo);
     });
   }
 
   function getFileName(path) {
     return path.split('/').pop();
   }
+
+  function setFieldVisibility(id, fileinfo) {
+    // Field label in info view
+    var label = $(id).previousElementSibling;
+
+    // Check for video file and hide respective field label and
+    // its value and remove bottom border from the previous field.
+    if (fileinfo.metadata.video) {
+      $(id).classList.add('hide');
+      label.classList.add('hide');
+      label.previousElementSibling.classList.add('no-border');
```

Comment:
Note that you only want the no-border thing if the element you're hiding is the last one in the list.  So the way this is written it only works right for hiding the resolution.  Arguably that is good enough to fix this blocker.  But you might be able to do this with something like `if ($(id).nextElementSibling === null) { /* set the no-border thing */ }`